### PR TITLE
feat: configure transaction longevity

### PR DESCRIPTION
**Pull request by ftupas**
# Pull Request type

- Feature
- Code style update (formatting, renaming)
- Refactoring (no functional changes, no API changes)

## What is the current behavior?

Resolves: #539 

## What is the new behavior?

- Add `TransactionLongevity` to the pallet config so it can be configurable
- Set the longevity value in runtime
- Add tests for transaction longevity

## Does this introduce a breaking change?

No

### CODE REVIEW

**Review by LucasLvy — CHANGES_REQUESTED** [review]
Change the value otherwise lgtm

File: crates/runtime/src/pallets.rs [hunk 1]
```diff
@@ -184,6 +185,7 @@ impl pallet_sudo::Config for Runtime {
 
 parameter_types! {
     pub const UnsignedPriority: u64 = 1 << 20;
+    pub const TransactionLongevity: u64 = 64;
```
[the remark this answers is not in the document]
Comment (ftupas) [reply crates/runtime/src/pallets.rs:188]: Thanks for the review @LucasLvy ! Will update this

[the remark this answers is not in the document]
Comment (ftupas) [reply crates/runtime/src/pallets.rs]: Just a small question, would it be alright for this to be a bit high for longevity? Tx might never be dropped from the mempool...

[the remark this answers is not in the document]
Comment (LucasLvy) [reply crates/runtime/src/pallets.rs]: Yes for now this is what we want

**Review by LucasLvy — APPROVED** [review]
lgtm

**Pull request merged**